component/stream/read_write.go

```diff
@@ -326,15 +339,15 @@ func (rw *ReadWriteCache) Stop() error {
 	return nil
 }
 
-func (rw *ReadWriteCache) purge(handle *handlemap.Handle, size int64, flush bool) error {
+func (rw *ReadWriteCache) purge(handle *handlemap.Handle, size int64) error {
 	handle.CacheObj.Lock()
 	defer handle.CacheObj.Unlock()
-	if flush {
-		err := rw.NextComponent().FlushFile(internal.FlushFileOptions{Handle: handle})
-		if err != nil {
-			return err
-		}
-	}
+	// if flush {
```

Comment:
As this flag itself is removed from function call, we can remove this dead code.

Reply:
sure